manifests/compute/rackspace.pp

```diff
@@ -2,7 +2,7 @@
 #
 # Provides support for Rackspace compute resources
 #
-# === Parameters:
+# === Advanced parameters:
```

Comment:
We should kill this file. The package was dropped in 2.1.

Reply:
Happening here -- https://github.com/theforeman/puppet-foreman/pull/894